Randomizer.cs

```diff
@@ -1182,6 +1182,30 @@ private void PlaceFreeItems(List<int> itemPool)
                 ItemList[freeItem].ReplacesItemId = Items.SongHealing;
                 itemPool.Remove(Items.SongHealing);
             }
+            if (_settings.StartingRemains > 0)
```

Comment:
I believe this can be simplified by doing something like this:

    List<(string name, byte mask)> remains = new List<(string, byte)>() 
    {
        ("Odolwa",  0x01),
        ("Goht",    0x02),
        ("Gyorg",   0x04),
        ("Twinmold", 0x08)
    };

    byte startingRemains = 0;
    for (int i = 0; i < _settings.StartingRemains; i++)
    {
        int j = _random.Next(remains.Count);
        var pick = remains[j];
        remains.RemoveAt(j);
        Debug.WriteLine($"Given {pick.name} Remains");
        startingRemains |= pick.mask;
    }
    _randomized.StartingRemains = startingRemains;

this also changes it so that you are actually given N remains, instead of a random amount between 1 and N